ffs/api/types.go

```diff
@@ -26,25 +26,52 @@ type InstanceStore interface {
 // Config has general information about a Api instance.
 type Config struct {
 	ID               ffs.APIID
-	WalletAddr       string
+	Addrs            map[string]AddrInfo
 	DefaultCidConfig ffs.DefaultCidConfig
 }
 
+type AddrInfo struct {
+	Name string
+	Addr string
+}
+
 // InstanceInfo has general information about a running Api instance.
 type InstanceInfo struct {
 	ID               ffs.APIID
 	DefaultCidConfig ffs.DefaultCidConfig
-	Wallet           WalletInfo
+	Wallet           []BalanceInfo
```

Comment:
Nit: Naming seems weird. Should the attributes be called `Balances`? Or the type `WalletInfo`.

Reply:
I like the idea of `Balances` for the attribute